src/averageAges.js

```diff
@@ -20,6 +20,12 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const filteredMen = people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
+
+  return calculateAverageHelper(filteredMen);
```

Comment:
i'm surprised this even worked since you introduce the `calculateAverageHelper` function later with a const special word, but using it before declaration 🤔